test/Jamfile.v2

```diff
@@ -18,7 +18,7 @@
 
 # bring in rules for testing
 import testing ;
-import property-set ;
+import threadapi ;
```

Comment:
Does it mean that now any Boost library that needs Boost.Thread needs to import threadapi.jam?